docs/Translation.md

```diff
@@ -0,0 +1,78 @@
+---
+description: BTCPay Translation features.
+---
+
+# Using BTCPay Translation Feature to Localize Your BTCPay Server instance
+
+
+BTCPay Server now includes a powerful translation feature that allows merchant administrators manage their instance in their preferred language. 
+
+This guide will help you create and manage translations, making the BTCPay platform more user-friendly for you.
```

Comment:
Don't think info here is adequate for a tech doc, if we use ChatGPT let's try not to make it sound like a marketer. 
```suggestion
This guide will help you create and manage translations, making the BTCPay platform more user-friendly for you.
```